src/containters/Home/components/DiscountsList/index.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
```

Comment:
![Screen Shot 2021-05-26 at 9 24 30 PM](https://user-images.githubusercontent.com/11296684/119719685-5e6d5880-be69-11eb-8289-b6543d7b9a82.png)

-------
**UI Issues**

- Card height is less than the design and width is more than the design.
- Color of rate and rate number don't match the design.
- Font size of all texts needs to be more bigger than current values.
- Bottom shadow.
- Spacing between elements.